### ISSUE TITLE
 Failed to load the tool-bar package

### ISSUE THREAD

**Issue by Lwenli1224** [suda/tool-bar#221]
[Enter steps to reproduce:]

1. ...
2. ...

**Atom**: 1.23.3 x64
**Electron**: 1.6.15
**OS**: Mac OS X 10.13.3
**Thrown From**: [tool-bar](ssh://git@github.com/suda/toolbar) package 1.1.5

### Stack Trace

Failed to load the tool-bar package

```
At variable @button-text-color-selected is undefined in /Users/jasonli/.atom/packages/tool-bar/styles/tool-bar.less:37:13

LessError: variable @button-text-color-selected is undefined
  at /packages/tool-bar/styles/tool-bar.less:37:13
```

### Commands

```
```

### Non-Core Packages

```
1989-syntax 0.5.0 
activate-power-mode 2.7.0 
atom-beautify 0.30.9 
atom-ide-ui 0.8.1 
atom-jshint 2.0.0 
atom-live-server 2.2.0 
atom-material-syntax 1.0.8 
atom-material-ui 2.1.1 
atom-react-autocomplete 0.1.10 
atom-ternjs 0.18.3 
atom-visual-studio-code-ui 1.3.4 
autoclose-html 0.23.0 
autocomplete-html-entities 0.2.0 
autocomplete-js-import 1.4.0 
autocomplete-paths 2.12.2 
autoprefixer 3.7.1 
block-comment 0.4.3 
browser-plus 0.0.98 
busy-signal 1.4.3 
chester-atom-syntax 0.3.0 
chester-kh-syntax 0.4.2 
color-picker 2.2.5 
comment 0.6.1 
csslint 1.2.0 
darkula-ui 0.1.0 
docblockr 0.13.6 
emmet 2.4.3 
emmet-jsx-css-modules 1.0.0 
evangelism-syntax 0.13.0 
file-icons 2.1.16 
firewatch-syntax 1.1.1 
git-control 0.9.0 
git-plus 7.10.0 
gloom 0.6.5 
goto-definition 1.3.4 
hyperclick 0.0.0 
ide-typescript 0.7.2 
improved-chester-atom-syntax 0.3.0 
intellij-idea-keymap 0.2.3 
javascript-snippets 1.2.1 
jquery-snippets 11.0.0 
js-hyperclick 1.13.0 
jshint 1.8.6 
language-babel 2.83.1 
language-javascript-jsx 0.3.7 
language-stylus 0.2.0 
language-vue 0.23.1 
linter 2.2.0 
linter-csslint 2.0.0 
linter-eslint 8.4.1 
linter-jshint 3.1.6 
linter-ui-default 1.6.10 
minimap 4.29.7 
minimap-quick-highlight 1.0.1 
monokai 0.24.0 
node-debugger 1.10.1 
nord-atom-ui 0.11.0 
one-dark-vivid-syntax 1.8.0 
one-sharp-syntax 3.0.0 
open-html-in-browser 0.2.1 
open-in-browser 0.5.2 
package-sync 1.1.0 
pdf-view 0.65.0 
pigments 0.40.2 
platformio-ide-terminal 2.8.0 
prettier-atom 0.50.0 
project-manager 3.3.5 
quick-highlight 0.12.0 
react-es6-snippets 0.3.0 
react-redux-atom-snippets 0.7.2 
react-snippets 1.0.1 
script 3.17.3 
set-syntax 0.4.0 
seti-syntax 1.1.3 
seti-ui 1.10.0 
Stylus 3.1.1 
sublime-block-comment 0.5.1 
sync-settings 0.8.3 
tool-bar 1.1.5 
tool-bar-main 0.0.11 
vue-element-ui-snippets 0.7.0 
vue2-autocomplete 0.1.0-alpha.4 
vuejs2-snippets 0.15.0 
```

**Comment by edreesen** [comment]
I have the same issue, this should be a duplicate of #219.

**Comment by Tbfaldhi** [comment]
i have error message:
[Enter steps to reproduce:]

1. ...
2. ...

**Atom**: 1.23.3 x64
**Electron**: 1.6.15
**OS**: Ubuntu 17.10
**Thrown From**: [tool-bar](ssh://git@github.com/suda/toolbar) package 1.1.5


### Stack Trace

Failed to load the tool-bar package

```
At variable @button-text-color-selected is undefined in /home/faldhifal/.atom/packages/tool-bar/styles/tool-bar.less:37:13

LessError: variable @button-text-color-selected is undefined
  at /packages/tool-bar/styles/tool-bar.less:37:13
```

### Commands

```
```

### Non-Core Packages

```
atom-beautify 0.30.9 
autocomplete-clang 0.11.4 
build 0.70.0 
busy-signal 1.4.3 
color-picker 2.2.5 
css-snippets 1.1.0 
emmet 2.4.3 
es6-snippets 0.3.0 
file-icons 2.1.16 
intentions 1.1.5 
language-ini 1.19.0 
linter 2.2.0 
linter-gcc 0.7.1 
linter-ui-default 1.6.10 
platformio-ide 2.0.1 
platformio-ide-debugger 1.2.5 
platformio-ide-terminal 2.8.0 
script 3.17.3 
tool-bar 1.1.5 
```

**Comment by suda** [comment]
Duplicate of https://github.com/suda/tool-bar/issues/219. Fixed in 1.1.6
